tests/test_expressions.py

```diff
@@ -791,6 +791,7 @@ def test_properties_from_dict(self):
 
     def test_convert(self):
         from collections import namedtuple
+        from zoneinfo import ZoneInfo
```

Comment:
This was [added](https://docs.python.org/3/library/zoneinfo.html) in python 3.9, so it's gonna be problematic when running 3.7 and 3.8. Is there any other way to do it?

Reply:
yeah, I just realised :) switching it to pytz now

Reply:
I put the `zoneinfo` test behind a version gate and used `pytz` in the main test instead.

As an aside, is there a reason why we still support Python 3.7? The upstream Python project [doesnt even support](https://devguide.python.org/versions/) Python 3.7

Reply:
Because some folks are still using sqlglot with python 3.7 due to other packages that need this version. We did discuss getting rid of it at some point, but not sure if/when that will happen.